src/commands/deleteResourceGroup.ts

```diff
@@ -23,11 +23,13 @@ export async function deleteResourceGroup(context: IActionContext, primaryNode?:
 
     const deleteConfirmation: string | undefined = settingUtils.getWorkspaceSetting('deleteConfirmation');
     for (const node of selectedNodes) {
+        const numOfResources = await node.getNumOfResources(context);
+
         if (deleteConfirmation === 'ClickButton') {
-            const areYouSureDelete: string = localize('areYouSureDelete', 'Are you sure you want to delete resource group "{0}" and all its resources?', node.name);
+            const areYouSureDelete: string = localize('areYouSureDelete', 'Are you sure you want to delete resource group "{0}". This will delete all {1} resource(s) in the resource group?', node.name, numOfResources);
```

Comment:
The one with 1 resource makes sense but the one with 0 resources might get a little confusing. 
We might need some logic just for that.

Reply:
What if we go similar to the portal:
<img width="595" alt="Screen Shot 2021-05-11 at 10 15 05 AM" src="https://user-images.githubusercontent.com/11282622/117857640-d7519b00-b241-11eb-9489-77e88a5fe69d.png">

I feel like that sentence would make sense even for 0.

Reply:
Also, either way, the question mark should go on the "Are you sure" sentence, not the second one.

Reply:
Yeah, I like the way that looks quite a bit better.  Though it doesn't read correctly if there is 1 resource.

![image](https://user-images.githubusercontent.com/5290572/117866848-767b9000-b24c-11eb-8643-ba2b8a56c969.png)

![image](https://user-images.githubusercontent.com/5290572/117866877-7da29e00-b24c-11eb-8a31-1cf398233899.png)

It's the kind of thing that kind of bothers me enough to have logic for it, but I dunno 😒